### PR TITLE
[doc] Add 2024 Video Links

### PR DESCRIPTION
**Pull request by KaiSernLim**
## Summary

Updating the docs by adding some 2024 links, especially the podcast interview with The Geek Narrator and the QCon talk.

## Does this PR introduce any user-facing changes?

- [X] No. You can skip the rest of this section.

### CODE REVIEW

File: docs/user_guide/learn_more.md [hunk 1]
```diff
@@ -22,20 +22,22 @@ The following blog posts have previously been published about Venice:
...
 
-- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE)
+- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE) # TODO: private video?
 - 2019: [People You May Know: Fast Recommendations over Massive Data](https://www.infoq.com/presentations/recommendation-massive-data/)
 - 2019: [Enabling next generation models for PYMK Scale](https://www.youtube.com/watch?v=znd-Q6IvCqY)
 - 2022: [Open Sourcing Venice](https://www.youtube.com/watch?v=pJeg4V3JgYo)
 - 2023: [What is Derived Data? (and Do You Already Have Any?)](https://www.infoq.com/presentations/derived-data/)
 - 2023: [Partial Updates in Venice](https://www.youtube.com/watch?v=WlfvpZuIa6Q&t=3880s)
-- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=yVXtWzAbcmY)
+- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=yVXtWzAbcmY) # TODO: Is this the right link?
```
Comment (KaiSernLim) [inline docs/user_guide/learn_more.md:40]: Are these two links correct?

Comment (nisargthakkar) [reply docs/user_guide/learn_more.md:40]: https://www.youtube.com/watch?v=jfbg6IUgVlI

Comment (FelixGV) [reply docs/user_guide/learn_more.md:40]: Good catch... thanks!

File: docs/user_guide/learn_more.md [hunk 2]
```diff
@@ -22,20 +22,22 @@ The following blog posts have previously been published about Venice:
...
+- 2024: [Stable Release 0.4.263](https://blog.venicedb.org/stable-release-0-4-263)
 
 Follow us on [HashNode](http://blog.venicedb.org) or [subscribe to the newsletter](https://blog.venicedb.org/newsletter) 
 to hear about it when new posts are published.
 
 ## Talks
 The following talks have been given about Venice:
 
-- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE)
+- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE) # TODO: private video?
```
Comment (FelixGV) [inline docs/user_guide/learn_more.md:34]: Looks like that video is gone indeed... let's comment out the whole line. We can always uncomment it later if we figure out how to get to it again.

**Comment by gaojieliu** [comment]
@KaiSernLim 
Since you are updating this , there is one talk:
https://www.infoq.com/presentations/scalable-low-latency/

File: docs/user_guide/learn_more.md [hunk 3]
```diff
@@ -22,20 +22,23 @@ The following blog posts have previously been published about Venice:
...
-- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE)
+[//]: # (- 2018: [Venice with Apache Kafka & Samza]&#40;https://www.youtube.com/watch?v=Usz8E4S-hZE&#41;)
 - 2019: [People You May Know: Fast Recommendations over Massive Data](https://www.infoq.com/presentations/recommendation-massive-data/)
 - 2019: [Enabling next generation models for PYMK Scale](https://www.youtube.com/watch?v=znd-Q6IvCqY)
 - 2022: [Open Sourcing Venice](https://www.youtube.com/watch?v=pJeg4V3JgYo)
 - 2023: [What is Derived Data? (and Do You Already Have Any?)](https://www.infoq.com/presentations/derived-data/)
 - 2023: [Partial Updates in Venice](https://www.youtube.com/watch?v=WlfvpZuIa6Q&t=3880s)
-- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=yVXtWzAbcmY)
+- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=jfbg6IUgVlI)
+- 2024: [The Journey to a Million Ops / Sec / Node in Venice](https://www.infoq.com/presentations/scalable-low-latency/)
```
Comment (gaojieliu) [inline docs/user_guide/learn_more.md:41]: BTW, this happened in 2023.

Comment (KaiSernLim) [reply docs/user_guide/learn_more.md]: Fixed. I guess the March 16, 2024 in the link was just the date the recording was published.

**Review by gaojieliu — APPROVED** [review]
Thanks!

**Pull request merged**